server/models/Request.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+    {
+        ownerId: {
+            type: mongoose.Schema.Types.ObjectId,
+            required: true,
+            ref: 'User'
+        },
+        ownerName: {
+            type: String,
+            required: true,
+            unique: true
+        },
+        ownerPhoto: {
+            type: String,
+            required: true,
+        },
+        sitterId: {
+            type: mongoose.Schema.Types.ObjectId,
+            required: true,
+            ref: 'User'
+        },
+        sitterName: {
+            type: String,
+            required: true,
+            unique: true
+        },
+        location: {
+            type: String,
+            default: '',
+        },
+        start: {
+            type: Date,
+            required: true,
+            default: Date.now,
+        },
+        end: {
+            type: Date,
+            required: true,
+            default: () => Date.now() + 24*60*60*1000,
+        },
+        animalType: {
+            type: String,
+            enum: ['dog', 'cat', 'other'],
+            required: true,
+        },
+        usefulInfo: {
+            type: String,
+            default: '',
+        },
+        accepted: {
```

Comment:
Additionally, we can consolidate the `accepted/declined` fields under a single enum type object.

Example: 
```
status: {
    type: String,
    enum: ['pending', 'accepted', 'declined', 'completed']
    default: 'pending'
    lowercase: true,
    required: true,
  }
```